suite-native/navigation/src/components/ScreenSubHeader.tsx

```diff
@@ -0,0 +1,47 @@
+import { ReactNode } from 'react';
+
+import { Box, Text } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { GoBackIcon } from './GoBackIcon';
+
+type ScreenSubHeaderProps = {
+    content?: ReactNode;
+    rightIcon?: ReactNode;
+    leftIcon?: ReactNode;
+};
+
+const headerStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    padding: utils.spacings.small,
+    backgroundColor: utils.colors.backgroundSurfaceElevationNegative,
+    height: 48 + utils.spacings.small * 2,
+}));
+
+const ICON_SIZE = 48;
+const iconWrapperStyle = prepareNativeStyle(() => ({
+    width: ICON_SIZE,
+    height: ICON_SIZE,
+}));
+
+export const ScreenSubHeader = ({ content, rightIcon, leftIcon }: ScreenSubHeaderProps) => {
```

Comment:
Why `Sub`?

Reply:
Currently, the design counts with 2 types of headers. One with the device manager and one, that is just redesigned previous header. 

I like to keep the top (main) header name as `ScreenHeader` with the other (accompanying) header as SubHeader. Similar to Title and SubTitle

Reply:
https://m1.material.io/components/subheaders.html#subheaders-menu-subheaders

Reply:
The naming is not new and is used quite a bit in design systems so I thought it to be good fit